src/windows/port.cc

```diff
@@ -55,6 +55,8 @@ int safe_vsnprintf(char *str, size_t size, const char *format, va_list ap) {
   return _vsnprintf(str, size-1, format, ap);
 }
 
+#if !(defined(_MSC_VER) && (_MSC_VER >= 1900))
+// MSVC 1900 defines this method body as part of stdio.h.
```

Comment:
We still need it for MSVC versions 1800 and below. Only 1900 has implemented it.